<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicketPurchaseWindow</class>
 <widget class="QMainWindow" name="TicketPurchaseWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slyther Airlines - Administration Panel</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="MapContainer">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>740</width>
         <height>455</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2"/>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="sideframe">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="ZoomButtons">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="ZoomOut">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>0</y>
            <width>21</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
         <widget class="QPushButton" name="ZoomIn">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>21</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="ControlPanel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Control Panel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <widget class="QComboBox" name="AirportsBox">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>50</y>
            <width>101</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>30</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Select Origin</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>100</y>
            <width>91</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Select Destination</string>
          </property>
         </widget>
         <widget class="QComboBox" name="AirportsBox_2">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>120</y>
            <width>101</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QScrollArea" name="scrollArea">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>230</y>
            <width>120</width>
            <height>171</height>
           </rect>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>118</width>
             <height>169</height>
            </rect>
           </property>
          </widget>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>180</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Cost: </string>
          </property>
         </widget>
         <widget class="QLabel" name="costLabel">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>180</y>
            <width>91</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>210</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Travel Path</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>963</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionBack_to_Airlines"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionBack_to_Airlines">
   <property name="text">
    <string>Back to Airlines</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
